content/docs/lists-and-keys.md

```diff
@@ -1,115 +1,114 @@
 ---
 id: lists-and-keys
-title: Lists and Keys
+title: Llistes i Claus
 permalink: docs/lists-and-keys.html
 prev: conditional-rendering.html
 next: forms.html
 ---
 
-First, let's review how you transform lists in JavaScript.
+Primer, revisem com es transformen les llistes a JavaScript.
 
-Given the code below, we use the [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function to take an array of `numbers` and double their values. We assign the new array returned by `map()` to the variable `doubled` and log it:
+En el codi que segueix a sota, utilitzem la funció [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) per prendre una matriu de `numeros` i duplicar-ne els seus valors. Assignem la nova array retornada per `map()` a la variable `doblat` i la mostrem a la consola:
 
 ```javascript{2}
-const numbers = [1, 2, 3, 4, 5];
-const doubled = numbers.map((number) => number * 2);
-console.log(doubled);
+const numeros = [1, 2, 3, 4, 5];
+const doblat = numeros.map((num) => num * 2);
+console.log(doblat);
 ```
 
-This code logs `[2, 4, 6, 8, 10]` to the console.
+Aquest codi mostra  `[2, 4, 6, 8, 10]` a la consola.
 
-In React, transforming arrays into lists of [elements](/docs/rendering-elements.html) is nearly identical.
+A React, transformar arrays en llistes d'[elements](/docs/rendering-elements.html) és gairebé idèntic.
 
-### Rendering Multiple Components {#rendering-multiple-components}
+### Renderització de múltiples components {#rendering-multiple-components}
 
-You can build collections of elements and [include them in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) using curly braces `{}`.
+Podem construir col·leccions d'elements i [incloure-les a JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) fent servir claus `{}`.
 
-Below, we loop through the `numbers` array using the JavaScript [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function. We return a `<li>` element for each item. Finally, we assign the resulting array of elements to `listItems`:
+A continuació, farem servir el mètode de Javascript [`map()`](https://developer.mozilla.org/ca/docs/Web/JavaScript/Reference/Global_Objects/Array/map) a l'array `numeros`. Aquest ens retornarà un element `<li>` per a cada element de l'array. Després, assignarem l'array resultant a la variable `llistadElements`
```

Comment:
```suggestion
A continuació, farem servir el mètode de Javascript [`map()`](https://developer.mozilla.org/ca/docs/Web/JavaScript/Reference/Global_Objects/Array/map) a l'array `numeros`. Aquest ens retornarà un element `<li>` per a cada element de l'array. Després, assignarem l'array resultant a la variable `llistadElements`:
```

Reply:
With respect to `llistadElements` I am not sure if it refers to "Llista d'elements" or "Llistat elements". In my opinion, the first case should be `llistaDElements` in camel case and the second one `llistatElements`. Please, feel free to propose any alternative if you think there's a better approach.

Reply:
I gave the name from (line 22):
`A React, transformar arrays en llistes d' **[elements]**(/docs/rendering-elements.html) és gairebé idèntic.`
I change all for `llistaDElements`.